clusters/app.ci/ocs/README.md

```diff
@@ -0,0 +1,10 @@
+# OCS Infrastructure
+
+## Generating an Image Pull Credential
+
+First, log in to [the cluster](https://api.ci.openshift.org/console/catalog). Then, run:
```

Comment:
This doc is outdated.
Because the cluster is app.ci.
https://docs.ci.openshift.org/docs/getting-started/useful-links/

You need to update console URL.

Reply:
updated